docs/gallery/transformations/plot_drop_event_by_area.py

```diff
@@ -0,0 +1,28 @@
+"""
+==========
+DropEventByArea
+==========
+The :class:`~tonic.transforms.DropEventByArea` removes
+all events located in a randomly chosen box area.
+"""
+
+import tonic
+
+nmnist = tonic.datasets.NMNIST("../../2_tutorials/data", train=False)
+events, label = nmnist[0]
+
+transform = tonic.transforms.Compose(
+    [
+        tonic.transforms.DropEventByArea(
+            sensor_size=nmnist.sensor_size, area_ratio=0.4
+        ),
+        tonic.transforms.ToFrame(
+            sensor_size=nmnist.sensor_size,
+            n_time_bins=20,
```

Comment:
just so that all the transforms in the gallery are rendered at the same speed, could you use time_window=10000 ?